jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PluginConfigurationProcessor.java

```diff
@@ -68,12 +74,25 @@ static AbsoluteUnixPath getAppRootChecked(JibExtension jibExtension) {
    * @throws InvalidImageReferenceException if parsing the base image configuration fails
    */
   static PluginConfigurationProcessor processCommonConfiguration(
-      JibLogger logger, JibExtension jibExtension, GradleProjectProperties projectProperties)
+      Logger logger, JibExtension jibExtension, GradleProjectProperties projectProperties)
       throws InvalidImageReferenceException, NumberFormatException {
     JibSystemProperties.checkHttpTimeoutProperty();
 
     // TODO: Instead of disabling logging, have authentication credentials be provided
-    GradleJibLogger.disableHttpLogging();
+    // Disables Apache HTTP client logging.
```

Comment:
Let's put this in a separate method since this is not really processing common configuration.